sdk/core/azure-core/src/main/java/com/azure/core/http/HttpRequest.java

```diff
@@ -77,6 +96,15 @@ public URL getUrl() {
         return url;
     }
 
+    /**
+     * Get the target address as a String.
+     *
+     * @return the target address
+     */
+    public String getUrlString() {
```

Comment:
I'm not a fan of this naming.

Reply:
no, we have this removed. Had initially added this thinking we should be consistent across our api's but then came to the conclusion that it would rather be useful for us to handle the `try..catch`  than the user doing it always.
And then `getUrlString` was just `url.toString()` which didn't seem to add much value.